jenkins/build.sh

```diff
@@ -42,9 +42,17 @@ nice -15 make -j8 install
 ctest .
 
 # compile examples using the installation
+# simple example using installation
 cd ${cwd}/example/simple
 mkdir -p build && cd build
 cmake .. -Dgt_c_bindings_DIR=${cwd}/install/lib/cmake
 nice -15 make -j8
 ./driver
 
+# simple example using FetchContent
+cd ${cwd}/example/simple_fetch_content
+mkdir -p build && cd build
+cmake ..
+nice -15 make -j8
```

Comment:
I thought, `nice` accepts the niceness-value only with `-n` and also documentation says this ... for some reason it seems to work without `n` too...

Reply:
I think I always use it without `-n`, I'll change it.